packages/server/test/authHelper.ts

```diff
@@ -84,21 +85,42 @@ export type BasicTestUser = {
   role?: ServerRoles
 } & Partial<UserRecord>
 
+const initTestUser = (user: Partial<BasicTestUser>): BasicTestUser => ({
+  name: faker.person.fullName(),
+  email: faker.internet.email(),
+  id: '',
+  ...user
+})
+
 /**
  * Create basic user for tests and on success mutate the input object to have
  * the new ID
  */
-export async function createTestUser(userObj: BasicTestUser) {
-  if (!userObj.password) {
-    userObj.password = 'some-random-password-123456789#!@'
+export async function createTestUser(userObj: Partial<BasicTestUser>) {
```

Comment:
This allows us to skip a step in user creation - no longer need to define a `const user: BasicTestUser` w/ various values first